### DISCUSSION TITLE
Dexcom new account

### DISCUSSION THREAD

**Discussion by Cheun1** [Navid200/xDrip#86] [Dexcom]
Good evening, I would like to report a problem with xdrip + and the new dexcom accounts that require an email address as the username. When use xdrip + with ** dexcom share follow * data sources and the username is an email address, it doesn't work. I tested both with an old account with a normal username example diabetetoto it works , with the e-mail address which is imposed by dexcom now for example toto@gmail.com it does not work. Thank you

**Comment by Navid200** [comment]
![249321817_4652849991425876_4994017694231375532_n](https://user-images.githubusercontent.com/51497406/139151762-3eee0082-df01-4db3-bc3e-f632ed13f4f5.jpg)

**Comment by Navid200** [comment]
@tolot27 Would you please advise?

**Comment by Navid200** [comment]
@Cheun1 Please open an issue: https://github.com/NightscoutFoundation/xDrip/issues
